src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -259,6 +259,20 @@ Apply to thead to make headers not bold.
     padding-top: 0;
 }
 
+.question-recommended-length {
```

Comment:
I see I see. The `teammatesCommon.css` can indeed by re-organized. For now, maybe put these somewhere near `.session-name-table-width` or other component-related rules